README.md

```diff
@@ -1,8 +1,8 @@
 # *Blazor moved to the ASP.NET Core repo*
 
-The [sources](https://github.com/aspnet/AspNetCore/tree/master/src/Components) and [issues](https://github.com/aspnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Afeature-components) for Blazor have moved to the [ASP.NET Core repo](https://github.com/aspnet/aspnetcore).
+The [sources](https://github.com/dotnet/AspNetCore/tree/master/src/Components) and [issues](https://github.com/dotnet/AspNetCore/issues?q=is%3Aissue+is%3Aopen+label%3Afeature-components) for Blazor have moved to the [ASP.NET Core repo](https://github.com/dotnet/AspNetCore).
```

Comment:
Note that we changed the case in the `dotnet` repo to be all lowercase, so consider changing to `https://github.com/dotnet/aspnnetcore`.